SourceS/miniwin.h

```diff
@@ -15,7 +15,12 @@
 #include <time.h>
 // For _rotr()
 #ifdef DEVILUTION_ENGINE
-#include <x86intrin.h>
+// Some platforms do not have a hardware _rotr()
+#	ifdef __x86__
+#		include <x86intrin.h>
+#	else
+#		define _rotr(a,b)	((a >> b) ^ (a << (32-b)))

```

Comment:
I've removed all usages of `_rotl` and `_rotr` in https://github.com/diasurgical/devilutionX/commit/2cc0c2cfbb3a63e02f3f7a8630c25143b204204a

x86intrin shouldn't be needed anymore